source/deployment/sso-saml-ldapsync.rst

```diff
@@ -52,3 +52,13 @@ When enabled, SAML synchronization with AD/LDAP occurs in phases:
 
  - If any attribute of the user has changed, that attribute is copied from the LDAP server and the user is marked as updated.
  - If the corresponding ``LdapSettings.EmailAttribute`` is not found, the user is assumed to be deleted from the LDAP server, and deactivated from Mattermost by setting the ``Users.DeleteAt`` field to a valid timestamp.
+ 
+Override SAML Data with AD/LDAP Data
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ 
+Alternatively, you can choose to override the SAML bind data with the AD/LDAP information. This will help ensure that new users are not created when the email address changes for a user. 
+
+1. Configure SAML with AD/LDAP synchronization as specified above.  
+2. If using the "Id Attribute", map the SAML ``Id Attribute`` on **System Console > SAML > Id Attribute**. To ensure existing user accounts do not get disabled in this process, ensure the SAML IDs match the LDAP IDs. 
```

Comment:
1. If they're not using the Id Attribute, what would they do then?
2. How is the mapping done?
3. How can they ensure the SAML and LDAP IDs match?

Reply:
Changes reflected in https://github.com/mattermost/docs/pull/2281